deploy/config/example_deploy.toml

```diff
@@ -18,3 +18,8 @@ environment = "dev"
 regex = '^(?P<bucket>.*?)/(?P<key>.*)$'
 substitution_string = '$key'
 storage.backend = 'S3'
+
+[resolvers.storage.keys]
+location = "SecretsManager"
+private_key = "htsget-rs/c4gh-private-key" # pragma: allowlist secret
```

Comment:
This is neater than my proposed ARN-based lookup I reckon...

Reply:
It works with ARNs too, just depends on what you put.